public/app/features/scenes/querying/SceneQueryRunner.test.ts

```diff
@@ -197,4 +198,64 @@ describe('SceneQueryRunner', () => {
       expect(queryRunner.state.data?.series[0].fields[1].values.toArray()).toEqual([6, 12, 18]);
     });
   });
+
+  describe('when running queries with time range', () => {
+    beforeEach(() => {
+      runRequest.mockClear();
+    });
+
+    it('should run queries with timeshift applied', async () => {
+      const timeRange = { from: '2022-12-30T07:40:56.983Z', to: '2022-12-30T08:40:56.983Z' };
+      const queryRunner = new SceneQueryRunner({
+        queries: [{ refId: 'A' }],
+        $timeRange: new SceneTimeRange(timeRange),
+        timeShift: '1h',
+      });
+
+      queryRunner.activate();
+
+      await new Promise((r) => setTimeout(r, 1));
+
+      const sentRequest = runRequest.mock.calls[0][1];
+      expect(sentRequest.timeInfo).toEqual(' timeshift -1h');
+      expect(sentRequest.range.from).toEqual(dateTime(timeRange.from).subtract(1, 'h'));
+      expect(sentRequest.range.to).toEqual(dateTime(timeRange.to).subtract(1, 'h'));
+    });
+
+    it('should hide time override info if required', async () => {
+      const timeRange = { from: '2022-12-29T07:40:56.983Z', to: '2022-12-29T08:40:56.983Z' };
+      const queryRunner = new SceneQueryRunner({
+        queries: [{ refId: 'A' }],
+        $timeRange: new SceneTimeRange(timeRange),
+        timeShift: '1h',
+        timeFrom: '1h',
+        hideTimeOverride: true,
+      });
+
+      queryRunner.activate();
+
+      await new Promise((r) => setTimeout(r, 1));
+
+      const sentRequest = runRequest.mock.calls[0][1];
+      expect(sentRequest.timeInfo).toEqual('');
```

Comment:
I know this is how it is implemented currently, but I'm not sure if we need this. I get the impression that the `hideTimeOverride` is a pure UI option that was unfortunately placed in the Query options. Do you remember anything about that @torkelo?